assets/migrations/generator/generatorConfig.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE generatorConfiguration PUBLIC "-//mybatis.org//DTD MyBatis Generator Configuration 1.0//EN" "http://mybatis.org/dtd/mybatis-generator-config_1_0.dtd">
 <generatorConfiguration>
-	<classPathEntry
-		location="/home/samuel/Projects/opensrp_server/opensrp-server-configs/assets/migrations/drivers/postgresql-42.2.1.jar" />
+	<!-- Replace <project_root> here with the path to your project's root directory -->
+	<classPathEntry location="<project_root>/configs/assets/migrations/drivers/postgresql-42.2.1.jar" />
 	<context id="core">
 		<!-- <plugin type="org.mybatis.generator.plugins.RowBoundsPlugin"></plugin> -->
 		<jdbcConnection connectionURL="jdbc:postgresql://localhost:5432/opensrp"
 			driverClass="org.postgresql.Driver" password="admin" userId="opensrp_admin" />
 		<javaModelGenerator targetPackage="org.opensrp.domain.postgres"
-			targetProject="opensrp-server-core" />
+			targetProject="src/main/java" />
 		<sqlMapGenerator targetPackage="org.opensrp.repository.postgres.mapper.xml"
-			targetProject="opensrp-server-core" />
```

Comment:
The generator should be run from the configs project and not the opensrp-server-core so that its possible to run it against a different project in the future